### PR TITLE
Changes webpack-dev-server config to NOT open a browser automatically

### PR DESCRIPTION
**Pull request by alexcreasy**
## Description
* Changes webpack-dev-server config to NOT open a browser automatically

## How Has This Been Tested?

* run `npm run start:dev` from frontend folder.

## Test Impact

None, dev config only change
## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by andrewballantyne** [comment]
Just occurs to me... I'm not sure we are clear enough how to get to the UI now that it does not open.

```
Prepping files...
  SRC DIR: /Users/andrewballantyne/code/others/odh-dashboard/frontend/src
  OUTPUT DIR: /Users/andrewballantyne/code/others/odh-dashboard/frontend/public
  PUBLIC PATH: /

<i> [webpack-dev-server] [HPM] Proxy created: /api  -> http://0.0.0.0:4000
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:4010/, http://[::1]:4010/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/andrewballantyne/code/others/odh-dashboard/frontend/public' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
```
It does say localhost:4010 -- but it's not super clear. I was looking over our contributing and README, and I may just missed it, but it doesn't say launch a browser to localhost:4010 🤔

**Comment by alexcreasy** [comment]
@christianvogt good point, I'll update the docs and also make the change to output to the console.

**Comment by alexcreasy** [comment]
@christianvogt @andrewballantyne I've pushed some updates, I've updated the README's to reflect this workflow. I've also added a log line to the console - it kind of got lost in with the web pack noise, so I added some colour so it sticks out:

console_show_dev_server

WDYT?

File: frontend/README.md
Comment (andrewballantyne) [inline frontend/README.md:1]: Recommend you delete this file. "This project was bootstrapped with Create React App" lol, time for _this_ page to go away 🙂

Comment (christianvogt) [reply frontend/README.md:1]: Yes let's delete this file. If anything update the root `README.md` instead or even `CONTRIBUTING.md`.

**Review by andrewballantyne — APPROVED** [review]
/unhold

That works very nicely. The message is even clearer if you don't have the tab open.

image

**Pull request merged**